epochStart/bootstrap/simpleMetaBlockInterceptor.go

```diff
@@ -0,0 +1,37 @@
+package bootstrap
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+)
+
+type simpleMetaBlockInterceptor struct {
+	marshalizer      marshal.Marshalizer
+	receivedHandlers []block.MetaBlock
```

Comment:
maybe we can have 2 maps here: one that is map[string]*block.Metablock and map[string][]p2pPeerID. First is the connection between hash and metablock ant the other hash and the peer ID's that originated the message. In this manner it becomes impossible to fake the message

Reply:
done. Used the 2 maps and created a blocking function which waits for a metablock to be received X times (the X is passed by the epoch start data provider based on a percentage of the number of connected peers)